Replicated/Replicated.h

```diff
@@ -49,6 +53,12 @@ class Replicated {
         PlayerDebug
     };
 
+
+    static const Vector4 PLATINUM;
```

Comment:
Same as above, static const is a bit of a code smell, use constexpr

Reply:
![7lud16](https://github.com/The-Newcastle-Gang/SpeedFun/assets/122457074/3bf905d9-1633-4c1f-ab8a-82eedc8ac51c)